### PR TITLE
Move away from unsupported router

### PR DESCRIPTION
**Pull request by steveri**
This is phase 1 of a plan to move away from an unsupported router---SMT---and replace it with one that currently handles onebit_bool as well as at least one version of harris.  Phase 1 is to simply use the serpent router so as to finally get onebit_bool back into the regular testing infrastructure (serpent works well on onebit_bool but poorly on harris). Phase 2 should follow very quickly, which is where we replace serpent with keyi's router, which is strictly better than serpent in every dimension...

I think everyone already knew of and was on board with this plan, let me know if you have comments etc. Thanks!

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -381,51 +381,49 @@ build/%_CGRA_out.raw: build/%_pnr_bitstream
...
+# 	# HACK: Input file name to inpurt port file name, also pre-processing input
+# 	# file for DELAY
+# 	cd build; python ../TestBenchGenerator/process_input.py $*.io.json $*_input.raw $(DELAY)
+# 
+# 	cd build; ./Vtop
+# 
+# 	# HACK: Output port file name to output file name, also post-processing
+# 	# output file for DELAY
+# 	cd build; python ../TestBenchGenerator/process_output.py $*.io.json $*_CGRA_out.raw $* $(DELAY)
+# endif
```
Comment (leonardt) [inline makefile]: Can we just remove this if we're no longer going to use it? We can always search in the commit history to find the old code if we want to revive it.

Comment (steveri) [reply makefile]: yes of course...i will take care of it.
meanwhile it looks like checks are failing now, so i will look into that as well

**Comment by steveri** [comment]
Checks are passing again.  I'll wait another day maybe for more comments to come in and then finalize.
Lenny: I made the requested change, feel free to recheck (or not) thanks!

**Comment by dillonhuff** [comment]
Looks good to me

**Comment by steveri** [comment]
Thanks guys!

**Pull request merged**